### PR TITLE
feat: autoupdate ic deps

### PR DESCRIPTION
**Pull request by NikolaMilosa**
A small script that will be run as a scheduled job to create a PR with new IC versions.

### CODE REVIEW

File: .github/workflows/update-dependencies.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   schedule:
     # * is a special character in YAML so you have to quote this string
```
Comment (LittleChimera) [inline .github/workflows/update-dependencies.yaml:6]:
```suggestion
```

File: ic-revisions.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+	"https://github.com/dfinity/ic.git": {
+		"commit": "7dee90107a88b836fc72e78993913988f4f73ca2",
+		"ref": "refs/heads/master" 
+	},
+	"https://github.com/dfinity/cdk-rs.git": {
+		"commit": "59795716487fbb8a9910ac503bcea1e0cb08c932",
+		"ref": "refs/heads/main"
+	}
+}
```
Comment (LittleChimera) [inline ic-revisions.json:10]:
```suggestion
}

```

Comment (NikolaMilosa) [reply ic-revisions.json:10]: This file is managed by the script, I don't think that its necessary to format it :D

Comment (NikolaMilosa) [reply ic-revisions.json:10]: What i mean is that it will automatically have a new line

**Review by LittleChimera — APPROVED** [review]

**Pull request merged**